### PR TITLE
fix pause resume background transcribe

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

**Comment by xquanluu** [comment]
Tested on jambonz.me

File: lib/session/call-session.js [hunk 1]
```diff
@@ -1059,6 +1059,11 @@ class CallSession extends Emitter {
    * @param {string} opts.transcribe_status - 'pause' or 'resume'
   */
   async _lccTranscribeStatus(opts) {
+    if (this.backgroundTaskManager.isTaskRunning('transcribe')) {
+      this.backgroundTaskManager.getTask('transcribe').updateTranscribe(opts.transcribe_status);
+    } else {
+      this.logger.info('CallSession:_lccTranscribeStatus - there is no background transcribe task');
```
Comment (davehorton) [inline lib/session/call-session.js:1065]: this will be by far the common case, and no need to log any thing

**Review by davehorton — CHANGES_REQUESTED** [review]

**Comment by davehorton** [comment]
@xquanluu should we merge this?

**Comment by xquanluu** [comment]
Yes, I tested this.

The question for you is should we allow LCC transcribe_status be able to stop both background transcribe and foreground transcribe together.

**Comment by davehorton** [comment]
> Yes, I tested this.
> 
> The question for you is should we allow LCC transcribe_status be able to stop both background transcribe and foreground transcribe together.

So right now it only pauses the background transcribe, right?

**Comment by xquanluu** [comment]
> > Yes, I tested this.
> > The question for you is should we allow LCC transcribe_status be able to stop both background transcribe and foreground transcribe together.
> 
> So right now it only pauses the background transcribe, right?

Right now, it's only pauses the foreground transcribe

**Comment by davehorton** [comment]
ah, ok I see.  Well, I think we should wait on this PR until we can make it pause both/all transcribes.  I think the common use case is that some PII data is being spoken and we wouldn't want it to appear on any transcripts

**Comment by xquanluu** [comment]
I see, If that's the case then the PR is ready for merging, this PR will pause and resume both background and foreground transcribe

**Pull request merged**